Gemfile.lock

```diff
@@ -481,3 +481,6 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   vcr
   webmock
+
+BUNDLED WITH
```

Comment:
I don't think you meant to commit this. I'll let it fly this time since it seems to be working ;)